.github/workflows/build.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       # Push events on the main branch
       - main
+      - CTIA-8-verify-builds
```

Comment:
Assume we should remove this before merging.

Reply:
Removed. this is the execution before the line is removed: https://github.com/hashicorp/consul/runs/6629909144?check_suite_focus=true